environment/environment.go

```diff
@@ -37,6 +37,12 @@ const (
 	DefaultK8sConfigPath = ".kube/config"
 )
 
+var (
+	//nolint
+	//go:embed charts/*/**
```

Comment:
Found no rule in linter for that case "path not resolved" but that particular glob is required for dot files embedding.